lib/task-data/tasks/boot-livecd.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Boot LiveCD',
+    injectableName: 'Task.Os.Boot.LiveCD',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        profile: 'boot-livecd.ipxe',
+        completionUri: 'renasar-ansible.pub',
```

Comment:
I didn't see you reference the completionUri in your corresponding on-http PR (https://github.com/RackHD/on-http/pull/61) , so how we notify the ORA the installation success?